extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/internal/http/WemoHttpCall.java

```diff
@@ -43,4 +43,34 @@ public static String executeCall(String wemoURL, String soapHeader, String conte
 
     }
 
+    public static String getEventServiceXML(String wemoURL) {
+
+        try {
+
+            String eventServiceURL = wemoURL + "/eventservice.xml";
+
+            String wemoCallResponse = HttpUtil.executeUrl("GET", eventServiceURL, 2000);
+            return wemoCallResponse;
+
+        } catch (Exception e) {
```

Comment:
I see, it has already be done above, but I don't think that catching `Exception` is a good thing.
Perhaps you could catch every `RuntimeExcpetion` and the checked exceptions that are really thrown.